products/sle/main.pm

```diff
@@ -726,19 +726,33 @@ elsif (get_var("QA_TESTSUITE")) {
     loadtest "qa_automation/install_test_suite";
     loadtest "qa_automation/execute_test_run";
 }
-elsif (get_var("XFSTESTS")) {
-    prepare_target;
-    if (check_var('XFSTESTS', 'installation')) {
-        loadtest "xfstests/install";
+elsif (get_var('XFSTESTS')) {
+    if (get_var('FSTESTS_SPVM')) {
+        load_boot_tests();
+        load_inst_tests();
+        load_reboot_tests();
```

Comment:
you can use `prepare_target` for spvm as well, like here https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/5b8f261dfe928aa79768f3a889e937e62afbe1b6/products/sle/main.pm#L1136

Reply:
If so, that would be better, let me try use `prepare_target`, and also change to use `is_spvm` to got a test. Thanks!